dev/cnf/resources/bnd/bundle.props

```diff
@@ -38,7 +38,7 @@ Bundle-Version: ${bFullVersion}.${if;${driver;gradle};${version.qualifier};eclip
 -consumer-policy :    ${range;[==,+)}
 
 # Require a 1.6 JDK (at least): see section 3.4.1 of OSGi 4.3 core spec
-Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

Comment:
This should only be incremented by 1 java version right? Java 6 --> 7.
Also, the comment on the line above needs to be updated.

Reply:
It should have been incremented to Java 7 before when we changed from Java 6 support being the default.  Currently we ship with Require-Capability of 6 in most of the Java 7 bundles.  The comment is still accurate.  It says at least.  We don't want someone setting the default lower than 6.  I can remove the comment though if you feel strongly about it.